rootfs/api/models/release.py

```diff
@@ -23,6 +23,7 @@ class Release(UuidAuditedModel):
     app = models.ForeignKey('App', on_delete=models.CASCADE)
     version = models.PositiveIntegerField()
     summary = models.TextField(blank=True, null=True)
+    failed = models.BooleanField(default=False)
```

Comment:
"failed" is a little ambiguous here. Would you mind adding a comment to describe that this is means it failed to deploy?